### DISCUSSION TITLE
Too Old and Too Large; challenges for an "early universe"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#185] [Astrophysics and Physics]
"Astronomers Found a 13 Million Light-Year Long Vine of Galaxies in the Ancient Cosmos"
[https://youtu.be/sKKoZllEQyY](https://youtu.be/sKKoZllEQyY)
Preprint {submitted to A&A}: [https://arxiv.org/abs/2311.04867](https://arxiv.org/abs/2311.04867)

**Comment by mikehelland** [comment]
FWIW, that channel isn't credible and certainly not affiliated with NASA.

**Reply by RedshiftDrift** [reply]
For less 'sensationalism' read the preprint {submitted to A&A} here: [https://arxiv.org/abs/2311.04867](https://arxiv.org/abs/2311.04867)

**Reply by bgreenman** [reply]
I support the arxiv/abs/2311.04867 paper

**Reply by bgreenman** [reply]
I think everyone in the alternative cosmology field should read the preprint

**Reply by OlliSant** [reply]
Very fine and important. That's what we are waiting! Make it scientific for the mainstream, if possible.

**Comment by RedshiftDrift** [comment]
"A galaxy only 350 million years old has surprising amounts of metal"
[https://phys.org/news/2023-11-galaxy-million-years-amounts-metal.html](https://phys.org/news/2023-11-galaxy-million-years-amounts-metal.html)
and the pre-print "JADES: Carbon enrichment 350 Myr after the Big Bang in a gas-rich galaxy"
[https://arxiv.org/abs/2311.09908](https://arxiv.org/abs/2311.09908)

**Reply by bgreenman** [reply]
Pre-print arxiv/2311.09908 and phys.org/news articles prove the Big Bang model is on its last legs unless foolish academics prop it up

**Comment by RedshiftDrift** [comment]
"JWST uncovers a surprising amount of nickel in adolescent galaxies"
*Researchers 'flabbergasted' by some of what they found in these young star factories.*
[https://www.astronomy.com/science/jwst-uncovers-a-surprising-amount-of-nickel-in-adolescent-galaxies/](https://www.astronomy.com/science/jwst-uncovers-a-surprising-amount-of-nickel-in-adolescent-galaxies/)

**Reply by bgreenman** [reply]
These new findings confirm my statement 4 days ago the Big Bang model is on its last legs

**Reply by OlliSant** [reply]
How to make the mainstream to understand that? We must collect all the arguments and make an alternative theory, or some good alternatives. Then defend those innovations in scientific papers.

Much work for everybody. And together.

**Reply by bgreenman** [reply]
A great idea would be to reference all the scientific papers that challenge the Big Bang model and have qualified researchers present these papers at physics and astonomy conferences

**Reply by OlliSant** [reply]
And philosophy conferences.

**Reply by bgreenman** [reply]
good to attend too

**Comment by RedshiftDrift** [comment]
"Candidate Galaxies at z ~ 11.3-21.8 and beyond: results from JWST's public data taken in its first year"
H. Yan et al., preprint [http://arxiv.org/abs/2311.15121](http://arxiv.org/abs/2311.15121)

The abstract contains the 'mandatory' disclaimer required for publication:
"The emergence of galaxy population at z ~ 17.3 or earlier is consistent with the suggestion of an early cosmic hydrogen reionization and is not necessarily a crisis of the ΛCDM paradigm."

"While no models in the pre-JWST era predicted the emergence of galaxy population at a time as early as z ~ 17.3, it is not necessarily a crisis for the ΛCDM paradigm if our candidates are proved to be at such high redshifts. For example, the new "feedback-free burst" model within the same paradigm can now produce abundant luminous galaxies in very early time, and it is possible to fine-tune to match our derived luminosity functions. The existence of a large number of galaxies at z ~ 17.3 is in fact consistent with the suggestion that the hydrogen reionization began at z ~ 17.2 based on the detection of global neutral hydrogen absorption signal. It is, however, in conflict with the determination of z_re = 7.64 based on the CMB anisotrophy measurements of the Planck mission."

- - -
The galaxy COSMOS_168 (photo p. 32) is quite impressive with a resolved ~0.5'' diameter.
SED fits give z = 18.4/16.8 (p. 51) and z = 20.7 (p. 54).
In a static universe, a diameter of [0.5'' at z = 16.8](https://cosmology.info/code/marmet_l/cosmology-calculators.html) corresponds to a physical diameter of ~28 kpc (like the Milky Way) at at a distance of 38 Gly.

**Reply by bgreenman** [reply]
H. Yan et al is looking for a way to prop up the Big Bang model, so he looking for an excuse by proping up the model by claiming that hydrogen reionization  took place at redshifts of 17.3 plus.

**Reply by HanDeBruijn** [reply]
From the [abstract](https://arxiv.org/abs/2311.15121):
>The candidates are selected as F150W, F200W and F277W dropouts, which correspond to z ~ 12.7 (11.3 < z < 15.4), 17.3 (15.4 < z < 21.8) and 24.7 (21.8 < z < 28.3), respectively.

**What** did they say ? z < 28.3 ? That's almost too close for comfort ! Because according to the Variable Mass Theory **z < 28.5 is a must**.
Unfortunately, much of the information in this article (and the previous) escapes me because of my English knowledge: I cannot grasp what the meaning is of "dropouts" (Dutch: "afvallers") in the context of these papers. Can someone ( @RedshiftDrift ) please help?

The link to [Cosmology Calculators](https://cosmology.info/code/marmet_l/cosmology-calculators.html) deserves a few remarks. Results of VMT (HdB version) are quite similar to those of SSTz Cosmology:
- The distance-redshift relation is exactly the same and it is good to read that _In a flat and static cosmology, the radial distance_ $r$ _is equal to the angular distance_ $d_A$.
- I think that the _bolometric luminosity distance_ is also the same as with SSTz, but I'm not sure.
- As can be checked with te bottom lines of [Let there be CMB](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm), the _Wien temperature_ and _spectral radiance enhanced_ are identical too.
- _The concept of "age of the universe" **has**_ a meaning in VMT, but only in the orbital clock time frame, not in the atomic clock  frame as "[measured](https://github.com/orgs/a-cosmology-group/discussions/160)" time.
### Update

The equation for calculating an [Escape velocity](https://en.wikipedia.org/wiki/Escape_velocity) is well known:

$$
\frac{1}{2}m v_e^2 = \frac{GMm}{r} \quad \Longrightarrow \quad v_e = \sqrt{\frac{2GM}{r}} 
$$

where $v_e=$ escape velocity, $GM=$ [Standard gravitational parameter](https://en.wikipedia.org/wiki/Standard_gravitational_parameter), $r=$ distance from the center of a body with mass $M$.

In a quasi-Newtonian setting, the basic equation for the expansion of the universe according to the Big Bang model is presented in the [Cosmology Slides](https://github.com/orgs/a-cosmology-group/discussions/189) - of course I have a backup somewhere :-)

![GR](https://github.com/a-cosmology-group/acg/assets/135433222/abe4f073-6666-4765-9e3b-e53941684c1f)

The standard expression for an escape velocity, with $v_e = dR_s/dt$ (: $R_s=$ [Schwarzschild radius](https://en.wikipedia.org/wiki/Schwarzschild_radius) ?), is recognized. But of course nothing can escape from an expanding universe except the boundary of the universe itself (sic!).

For a universe with $U=0$ a solution of the differential equation can immediately be found, assuming that $M_s=$ constant and $R_s(0)=0$.

$$
\frac{dR_s}{dt} = \sqrt{\frac{2GM_s}{R_s}} \quad \Longrightarrow \quad \frac{d(R_s)^{3/2}}{dt} = \frac{3}{2}(2GM_s)^{1/2} \quad \Longrightarrow \quad R_s = \left(\frac{9}{4}2GM_s\right)^{1/3} t^{2/3}
$$

corresponding with the "Matter alone" solution on [slide 47](https://github.com/orgs/a-cosmology-group/discussions/188#discussioncomment-7645966). Proceeding a little bit further we have two things:

![ook](https://github.com/a-cosmology-group/acg/assets/135433222/f344f171-1871-41d8-8089-8e699a07fd3a)
![nog](https://github.com/a-cosmology-group/acg/assets/135433222/ce7a363f-d08b-4aca-af0d-84603e20d280)

In a **flat** universe the density $\rho(t)$ is equal to the critical density $\rho_c(t)$ and so

$$
R_s = \left(\frac{9}{4}2GM_s\right)^{1/3} t^{2/3} = \left(\frac{9}{4}.G.\frac{8}{3}\pi R_s^3 \rho_c\right)^{1/3} t^{2/3} = R_s \left[\frac{3}{2} H(t) . t\right]^{2/3}
$$

$$
\mbox{Giving at last} \quad \left[\frac{3}{2} H(t) . t\right]^{2/3} = 1 \quad \Longrightarrow \quad t = \frac{2}{3H(t)}
$$

In the setting of a Static Euclidean Universe (SEU) the escape velocity is the speed of light ($v = c$) and the BB equation is replaced by the following equation as a starting point in [With other words](http://www.alternatievewiskunde.nl/cosmology/anders.htm).

$$
\frac{1}{2}mc^2 \gt \frac{GMm}{R}
$$

leading to a quite different theory altogether, but, in principle, with the same escape ;-)

**Reply by budrap00** [reply]
The standard model-fitting exercise, Why should this time be any different from all the other occasions the model has fallen on its face?

**Reply by bgreenman** [reply]
HanDeBrujin is right

**Reply by RedshiftDrift** [reply]
> What did they say ? z < 28.3 ?  I cannot grasp what the meaning is of "dropouts" (Dutch: "afvallers") in the context of these papers.

"Dropouts" means *"the galaxy disappears, or "drops out", from the picture"*[^1].  The images in the paper show galaxies as seen through different filters.  The filter number corresponds to the wavelength at the centre of the filter band, e.g. F150W is centered on $\lambda = 1500 {\rm \ nm}$, F200W is centered on $2000 {\rm \ nm}$, etc.

The galaxy is visible at longer wavelengths (images on the right).  The galaxy is no longer visible at shorter wavelengths (images on the left).  The reason is that light is absorbed by hydrogen at $\lambda < 121.5 {\rm \ nm}$ (Lyman-alpha) producing a "Lyman break" of the spectrum.  The image where the galaxy disappears (or *dropout image*) has the wavelength $\lambda = 121.5 {\rm \ nm}$ (as emitted by the galaxy) within the filter's band.

I calculated the corresponding redshift of the (approximate) transition wavelength between two filters.  For example, the galaxy COSMOS_168 (top set of images, shown below; filter numbers are indicated at the top of the image) disappears in the filter F200W, at a redshift approximately equal to $z = 2000 {\rm \ nm} / 121.5 {\rm \ nm} - 1 = 15.5$.

More precisely, the Lyman-alpha wavelength is redshifted by an amount corresponding to $13.4 \lesssim z \lesssim 18.6$.  Astronomers can use the shape of the spectra of known galaxies to interpolate/extrapolate, but the result becomes more model-dependent.

![COSMOS_168-dropout](https://github.com/a-cosmology-group/acg/assets/16748402/ada2d83a-8747-417c-a978-f9fc2073138f)

The galaxy smacs0723_B_001 (lower image) is barely visible through the F277W filter, meaning that it has a redshift almost reaching the upper limit of the filter band, corresponding to $z \simeq 25$.  This is the correct redshift if we are looking at a Lyman break galaxy (about 8% of bright galaxies [are Ly-break](https://arxiv.org/abs/1109.0228)).  32 galaxies were found in the F277W filter dropout, so statistically 3 of them are really at $z > 18.6$. Unfortunately the paper doesn't show the F356W dropout images (the $25 < z \simeq 28.3 < 30.5$ galaxies), but those are not very meaningful without a spectrum of emission lines.

In principle, a galaxy that drops out in the filter F410M can be seen with the Webb Telescope through the F444W filter: its deep-infrared detectors can see Lyman-break galaxies as far as $z = 34.1$.  However, the interpretation is not that simple and all galaxies are not clean "Lyman break".  There are other reasons that could produce absorption at longer wavelengths, and the astronomer's skills at identifying those reasons can contribute to a different interpretation of the images[^2].

Spectral lines are much more reliable than just using filters - the paper insists: *"Most objects in our sample are bright enough for JWST spectroscopic confirmation, and it is* ***imperative*** *to identify at least a fraction of them so that JWST’s pursuit of “first luminous objects” can be put onto a more solid footing."* The MIRI instrument on JWST can take a spectrum up to $\lambda = 12 \ \mu{\rm m}$.  The Lyman-alpha line at that wavelength corresponds to a redshift $z =12 \ \mu{\rm m}/121.5 {\rm \ nm} - 1 = 97$, but of course a single spectroscopic line would not be sufficient to establish that it is emitted by hydrogen! 

[^1]:  "Het sterrenstelsel verdwijnt uit beeld" - *star-system* is a nice word for a galaxy 👍.
[^2]:  JWST could be the telescope that breaks VMT if they don't waste too much time looking for aliens on exoplanets 👎.

**Reply by bgreenman** [reply]
Figures F200W And F277W are correct, because without a full spectrum of emission lines the exact redshift can't be calculated for particular galaxies.Some emission lines are absorbed by particular elements or compounds. the big question is will the JWST be used to break the VMT.Who is getting priority in using the JWST?

**Reply by HanDeBruijn** [reply]
@RedshiftDrift .

>JWST could be the telescope that breaks VMT [ .. ]

Really? Let's quote from [Hubble's law](https://en.wikipedia.org/wiki/Hubble%27s_law#Hubble_volume) (Wikipedia).
_Some cosmologists even use the term Hubble volume to refer to the volume of the [observable universe](https://en.wikipedia.org/wiki/Observable_universe), although this has a **radius approximately three times larger.**_ [bold by me]

Now the observable universe **in VMT** has a radius of $R_O = W.c/H$ where $W=3.383634283$ . Consequently, **our** $R_O$ is even **larger** than the _approximately three times_ the Hubble length in standard cosmology. So if you want to break VMT then you first have to break some Big Bang beliefs. That will represent a problem if your sources consist of papers describing observations within the mainstream paradigm.

**Reply by RedshiftDrift** [reply]
This quote is written from the perspective of the Big Bang model.  We know that distances [are incorrectly calculated in that paradigm](https://cosmology.info/code/marmet_l/).

**Reply by HanDeBruijn** [reply]
@RedshiftDrift . Just curious. How does **SSTz** prevent [Olbers’s paradox](https://en.wikipedia.org/wiki/Olbers%27s_paradox) ?

**Reply by RedshiftDrift** [reply]
Two aspects of the paradox must be explained:
1) **The bright sky**: As Olbers predicted, "any line of sight from Earth must end at the surface of a star and hence the night sky should be completely illuminated and very bright".  Since an increasing redshift with distance is observationally confirmed (this is not specific to SSTz) the night sky is not very bright at 5500 K ("white") but is very bright at 2.7 K, eliminating Olbers' *bright sky at visible wavelengths*.
2) **The energy density**: SSTz explains that the energy from the redshifted light is used to [produce fission of light elements](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7719752).  This imposes a limited amount of energy density that corresponds to the energy of the 2.7 K blackbody radiation.  This is compatible with observations, [as calculated here](https://en.wikipedia.org/wiki/Olbers%27s_paradox#Steady_state).[^1]

[^1]: The Big Bang hypothesis gives almost the same result. However, ΛCDM does not predict the correct angular distribution of the microwave background temperature, as demonstrated by the Hubble tension!

**Comment by RedshiftDrift** [comment]
1.2 Mly; "Structures larger than this size are incompatible with the [cosmological principle](https://en.wikipedia.org/wiki/Cosmological_principle) according to all estimates."
[https://en.wikipedia.org/wiki/List_of_largest_cosmic_structures#List_of_largest_structures](https://en.wikipedia.org/wiki/List_of_largest_cosmic_structures#List_of_largest_structures)

**Reply by HanDeBruijn** [reply]
![behoud](https://github.com/a-cosmology-group/acg/assets/135433222/2ca5203a-795a-4ca0-a750-ba74aac734fa)
It can be shown that **not** being homogeneous and isotropic is **not such a big deal** with an eternal and Static Euclidean Universe.

In the section about [Seeliger's Paradox](http://www.alternatievewiskunde.nl/cosmology/india.htm) at my website we find a possible [alternative](https://www.ptep-online.com/2018/PP-54-11.PDF) for the un[solved](http://www.alternatievewiskunde.nl/cosmology/Simeon.htm) integral in a [paper](https://www.ias.ac.in/article/fulltext/reso/021/05/0447-0452) by Amitabha Ghosh and Ujjal Dey. The latter works with spherical coordinates and the former works with cylindrical coordinates. After some necessary modifications the integral with cylindrical coordinates is:

$$
F = 2\pi G m \rho \int_{-\infty}^{+\infty} h \left[ \int_0^{\infty} \frac{e^{-\Gamma\sqrt{R^2+h^2}}R dR}{(R^2+h^2)^{3/2}}\right] dh
$$

where $m=$ test mass, $G=$ gravitational constant, $\rho=$ homogeneous and isotropic mass density, $\Gamma=H/c=$ with $H=$ Hubble parameter and $c=$ lightspeed.
With an analogous method as presented by [Simeon Albert L.M.](http://www.alternatievewiskunde.nl/cosmology/Simeon.htm) the outcome can be found: the force exerted on a test mass by the entire infinite universe is zero.

$$
F = 2\pi G m \rho \left[ - \frac{1}{2\Gamma} + \frac{1}{2\Gamma} \right] = 0
$$

Now suppose that, instead of infinite, our universe is semi-infinite. Formulated otherwise: our test mass is near the edge of an immensely large void below it and almost all the available mass of our universe is above it. Then we have, approximately, a non-zero acceleration $a_P$ towards the semi-infinite part of that hypothetical universe.

$$
F = 2\pi G m \rho \frac{1}{2\Gamma} \quad \Longrightarrow \quad a_P = \pi G \rho \frac{c}{H}
$$

According to formula (5.12) and the last sentences of section (6.2) in the book [_Origin of Inertia_](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/96836893X-rev.pdf) by Amitabha Ghosh we have

$$
\kappa = (\pi G_0 \rho)^{1/2} = H_0 \quad \Longrightarrow \quad \pi G \rho = H^2
$$

Notation has been adapted in our frame of reference because Amitabha thinks that there is a variable gravitational constant instead of a variable mass (density). Anyway with the above it follows that

$$
a_P = \pi G \rho \frac{c}{H} = H.c 
$$

Wikipedia is wrong. The [Pioneer anomaly](https://en.wikipedia.org/wiki/Pioneer_anomaly) is not explained by some _thermal recoil force_ but there is indeed a [cosmological connection](https://en.wikipedia.org/wiki/Pioneer_anomaly#Previously_proposed_explanations) and $a_P$ is not at all by coincidence close to Mordehai Milgrom's constant in [MOND](https://en.wikipedia.org/wiki/Pioneer_anomaly#Definition_of_gravity_modified) : $a_P = (8.74\pm 1.33)\times 10^{-10}m s^{-2} \approx a_0 \approx H.c = (6.59\pm 0.075)\times 10^{-10} m s^{-2}$ .

**Reply by RedshiftDrift** [reply]
It is trivial "that **not** being homogeneous and isotropic is **not such a big deal** in any" Non-Expanding Euclidean Universe.

**Comment by RedshiftDrift** [comment]
[Discovery of the Big Ring](https://astrodon.social/@pomarede@mastodon.social/111743319523681427), an ultra-large cosmological structure, by Alexia Lopez and her co-authors, further challenges what we understand about the universe.

This finding was announced yesterday at the Winter Meeting of the American Astronomical Society.

✅ AAS 243 Press Conference: [https://youtu.be/86Ps7vE6JHI?t=315](https://youtu.be/86Ps7vE6JHI?t=315)
"Do we need a new standard model?" ***Yes***

✅ Research News by UCLan: [https://www.uclan.ac.uk/news/big-ring-in-the-sky](https://www.uclan.ac.uk/news/big-ring-in-the-sky)
"A Big Cosmological Mystery" ***Not so in a non-expanding framework.***

✅ The Guardian: [https://www.theguardian.com/science/2024/jan/11/newly-discovered-cosmic-megastructure-challenges-theories-of-the-universe](https://www.theguardian.com/science/2024/jan/11/newly-discovered-cosmic-megastructure-challenges-theories-of-the-universe)
"Newly discovered cosmic megastructure challenges theories of the universe." ***It doesn't challenge non-expanding models of the universe.***

✅ BBC News: [https://www.bbc.com/news/science-environment-67950749](https://www.bbc.com/news/science-environment-67950749)
"Huge ring of galaxies challenges thinking on cosmos" ***but not everybody's thinking.***

**Comment by RedshiftDrift** [comment]
More challenges for the Big Bang model: too distant FRBs, galaxy structures that are too large, Hubble tension at 8 $\sigma$, galaxies going bananas, etc. [https://youtu.be/3NeKR7bqolY?t=440](https://youtu.be/3NeKR7bqolY?t=440)

Dr. Becky is going bananas too...
